osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySliderBall.cs

```diff
@@ -56,6 +62,8 @@ private void load()
                     Blending = BlendingParameters.Additive,
                 },
             };
+
+            slider.ApplyCustomUpdateState += updateStateTransforms;
```

Comment:
Other usages do this in `LoadComplete`, and also call once immediately:

https://github.com/ppy/osu/blob/f8830c6850128456266c82de83273204f8b74ac0/osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyMainCirclePiece.cs#L121-L125

Please follow existing conventions to ensure things work as expected.